packages/react/src/components/NumberInput/index.js

```diff
@@ -5,5 +5,10 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import NumberInput, { NumberInputView } from './NumberInput';
+
+NumberInput.View = NumberInputView;
```

Comment:
While there's no inherent problem in also tacking it on here, I think we need to pick a usage method for our documentation and stick with it.